Makefile

```diff
@@ -504,19 +268,19 @@ DEPS_PARTICLES = particles/particles.h particles/particles.cpp particles/field.h
 OBJS_PARTICLES = particles/physconst.o particles/particles.o particles/readfields.o particles/particleparameters.o particles/distribution.o readparameters.o version.o particles/scenario.o particles/histogram.o
 
 vlsvextract: ${DEPS_VLSVREADER} ${DEPS_VLSVREADERINTERFACE} tools/vlsvextract.h tools/vlsvextract.cpp ${OBJS_VLSVREADER} ${OBJS_VLSVREADERINTERFACE}
-	${CMP} ${CXXFLAGS} ${FLAGS} -c tools/vlsvextract.cpp ${INC_BOOST} ${INC_DCCRG} ${INC_EIGEN} ${INC_VLSV} -I$(CURDIR) 
+	${CMP} ${CXXFLAGS} ${FLAGS} -c tools/vlsvextract.cpp ${INC_BOOST} ${INC_DCCRG} ${INC_EIGEN} ${INC_VLSV} -I$(CURDIR)
 	${LNK} -o vlsvextract_${FP_PRECISION} vlsvextract.o  ${OBJS_VLSVREADERINTERFACE} ${LIB_BOOST} ${LIB_DCCRG}  ${LIB_VLSV} ${LDFLAGS}
 
 vlsv2silo:  ${DEPS_VLSVREADERINTERFACE} tools/vlsv2silo.cpp  ${OBJS_VLSVREADERINTERFACE}
```

Comment:
I think this is as good as dead, has anyone maintained it since the last umpteen changes to our VLSV format we did? I actually just dropped the mention on our installation guide the other day.

Reply:
I think I used it semi-recently still, but I agree it's a bit left n the dust. Might soon make sense to just dump it altogether and just use python where required.

Reply:
I used this just last December to check on a specific VDF of FHA which was reporting hitting the v-space wall. Downloading a single VDF .vlsv for a quick diagnostic was a lot easier than downloading a full restart file... So let's definately keep it in.

Reply:
Sorry I was not clear and the comment interface isn't showing where I actually clicked: I meant ditching vlsv2silo, definitely not vlsvextract, I'm a big fan of the latter!

Reply:
Ah, true. I don't believe anyone has used that silo export in the last 5 (10?) years. But let's make that a separate PR.